### ISSUE TITLE
The continue watching is not working

### ISSUE THREAD

**Issue by Cabowski92** [cranci1/Ryu#10]
The continue watching is not working

**Comment by cranci1** [comment]
If you are using HiAnime then yes is normal, i haven't got any time to add it so it will come in a little bit. (max 2/3 days, so next testflight build hopefully)

**Comment by Cabowski92** [comment]
Watching anime with gogo anime is not working with the default player

https://github.com/user-attachments/assets/bdc6846c-54e1-4045-a471-5576d1617db7

**Comment by cranci1** [comment]
You need to give the app a little bit. It will either start playing the video after a bit or close the loading page. In the second case that mean that there where an issue.

**Comment by Cabowski92** [comment]
> You need to give the app a little bit. It will either start playing the video after a bit or close the loading page. In the second case that mean that there where an issue.
> 
> If the loading view is then closed you can try changing the "Tries" in settings -> Sources Preferences -> Number of Retries (set it to the max)

I did all this and it does it indefinitely and won’t let you stop the process without clearing the app and opening it up again

**Comment by cranci1** [comment]
Is the view loading screen closing or is stuck there with no loading indicator after a bit?

**Comment by Cabowski92** [comment]
It’s stuck there until I completely clear the app

**Comment by cranci1** [comment]
can you please send a video until the loading screen is dismissed or the loading wheel stopped?

**Comment by Cabowski92** [comment]
That’s the thing neither of those things happen until the app is cleared from the background it stays their indefinitely the longest I let it run to see if it would clear was an hour

**Comment by cranci1** [comment]
can you please send me the backup file?

**Comment by Cabowski92** [comment]
Yes  did you get it when I replied to the email from GitHub?

**Comment by Cabowski92** [comment]
If not here it is again. 
[02_September_2024-1015.zip](https://github.com/user-attachments/files/16841375/02_September_2024-1015.zip)

**Comment by cranci1** [comment]
Thanks, the thing is that is working perfectly fine for me. 


https://github.com/user-attachments/assets/5e30f5f8-9e15-4119-bf4a-ec17e9d1814e


Is the loading screen just stuck there loadinf right?

**Comment by Cabowski92** [comment]
Yeah it’s still  getting stuck. I am on iPhone 12 Pro Max if that helps. 

https://github.com/user-attachments/assets/18095cc8-0332-4680-9403-d22c4b55eedf

![IMG_5365](https://github.com/user-attachments/assets/2437a03a-82b2-441a-9dd2-e35134f87990)

**Comment by Cabowski92** [comment]
So it looks like it’s the English translated anime websites beside hianime because it’s the only one that will play and not infinite load

**Comment by cranci1** [comment]
Even on new testflight build?

**Comment by Cabowski92** [comment]
Yes I even tried deleting and redownloading the app
